source/agora/common/Block.d

```diff
@@ -37,6 +38,45 @@ public struct BlockHeader
     /// The hash of the only transaction in the block
     /// (later to be replaced with a merkle root)
     public Hash tx_hash;
+
+
+    /***************************************************************************
+
+        Implements hashing support
+
+        Params:
+            dg = hashing function
+
+    ***************************************************************************/
+
+    public void computeHash (scope HashDg dg) const nothrow @safe @nogc
+    {
+        dg(this.prev_block[]);
+        hashPart(this.height, dg);
+        dg(this.tx_hash[]);
+    }
+}
+
+/// hashing test
+unittest
+{
+    import agora.common.crypto.Key;
+    auto address = `GDD5RFGBIUAFCOXQA246BOUPHCK7ZL2NSHDU7DVAPNPTJJKVPJMNLQFW`;
+    PublicKey pubkey = PublicKey.fromString(address);
+
+    // above parts not @safe/@nogc yet
+    () @safe @nogc nothrow
+    {
+        Output[1] outputs = [Output(100, pubkey)];
```

Comment:
```suggestion
        Output[1] outputs = [ Output(100, pubkey) ];
```

If you want to keep it consistent? The style guide does not cover this.

Reply:
I'm Switzerland on this one.